config/routes.rb

```diff
@@ -11,7 +11,7 @@
   namespace :api do
     namespace :v1 do
       resources :brands, only: [:index, :show]
-      resources :pizzas, only: [:show]
+      resources :pizzas, only: [:show, :new, :create]
```

Comment:
Unless im mistaken, this feature doesnt add a new pizza, it adds a new review. You should be adding to `reviews` routes, and a `reviews` controller, not pizzas.